openrtb/settlement.go

```diff
@@ -0,0 +1,10 @@
+package openrtb
+
+// Settlement is the interface that provides auction settlement details.
+//
+// Exchanges should implement this interface on the relevant auction detail objects.
+//
+// Price returns the auction settlement price.
+type Settlement interface {
```

Comment:
I wonder what should go into an auction settlement. Right now you only have `Price` which is good, we can extend this in the future easily. Given our use case right now, I wonder if it's worth the effort to include all macro values to be part of the settlement. Alternatively, we can use the `winner` object as a reference that @xiaopeng0216 is working on https://docs.google.com/spreadsheets/d/1GDLXin3If2jxMLTeRxdxfLHZMiFfZTawBterzUwMRts/edit#gid=0